README.md

```diff
@@ -27,64 +27,28 @@ Also see [buildah-build](https://github.com/redhat-actions/buildah-build) for mo
 
 Once an image has been built, [push-to-registry](https://github.com/redhat-actions/push-to-registry) can be used to push it to an image registry.
 
+<a id="action-inputs"></a>
+
 ## Action Inputs
 
-<table>
-  <thead>
-    <tr>
-      <th>Input</th>
-      <th>Required</th>
-      <th>Default</th>
-      <th>Description</th>
-    </tr>
-  </thead>
-
-  <tr>
-    <td>builder_image</td>
-    <td>Yes</td>
-    <td>-</td>
-    <td>
-      The location of the S2I builder image. A curated list of builder images can be found
-      <a href="./builder-images.md">here</a>.
-    </td>
-  </tr>
-
-  <tr>
-    <td>image_name</td>
-     <td>Yes</td>
-    <td>-</td>
-    <td>The name of the image to produce. </td>
-  </tr>
-
-  <tr>
-    <td>image_tag</td>
-    <td>No</td>
-    <td>latest</td>
-    <td>The tag of the image to produce.</td>
-  </tr>
-
-  <tr>
-    <td>path_context</td>
-    <td>No</td>
-    <td><code>.</code></td>
-    <td>The location of the path to run S2I from. This should be the path where your source code is stored.</td>
-  </tr>
-
-  <tr>
-    <td>log_level</td>
-    <td>No</td>
-    <td>1</td>
-    <td><a href="https://github.com/openshift/source-to-image/blob/master/docs/cli.md#log-levels">Log level</a> when running S2I. Can be 0 (least verbose) to 5 (most verbose).</td>
-  </tr>
-
-  <tr>
-    <td>env_vars</td>
-    <td>No</td>
-    <td>-</td>
-    <td>List of environment variable key-value pairs to pass to the s2i builder context. (eg. <code>key=value</code>, <code>mysecret=${{ secrets.MY_SECRET }}</code> ).</td>
-  </tr>
-
-</table>
+| Input Name | Description | Default |
+| ---------- | ----------- | ------- |
+| builder_image | The path of the S2I builder image. A curated list of builder images can be found [here](./builder-images.md). | **Required**
+| env_vars | List of environment variable key-value pairs to pass to the S2I builder context. (eg. `key=value`, `mysecret=${{ secrets.MY_SECRET }}`). | None
+| image | Name to give to the output image. | **Required**
+| tags | The tags of the image to build. For multiple tags, separate by a space. For example, `latest ${{ github.sha }}` | `latest`
+| log_level | [Log level](https://github.com/openshift/source-to-image/blob/master/docs/cli.md#log-levels) when running S2I. Can be 0 (least verbose) to 5 (most verbose). | `1`
+| path_context | The location of the path to run S2I from. This should be the path where your source code is stored. | `.`
+
+<a id="outputs"></a>
+
+## Action Outputs
+
+`image`: The name of the built image.<br>
+For example, `spring-image`.
+
+`tags`: A list of the tags that were created, separated by spaces.<br>
+For example, `latest ${{ github.sha }}`.
```

Comment:
i know this is copied from buildah-build, but this is a pretty poor example, since `github.sha` will be expanded into an actual commit ID, the output will never actually look like this. 

so we should replace this with a made-up commit hash or literally any other version-like string.